x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/layout/policy_event_filters_layout.tsx

```diff
@@ -118,6 +123,16 @@ export const PolicyEventFiltersLayout = React.memo<PolicyEventFiltersLayoutProps
             </EuiText>
           </EuiPageHeaderSection>
         </EuiPageHeader>
+        <EuiSpacer size="l" />
+        <EuiPageContent
+          hasBorder={false}
+          hasShadow={false}
+          paddingSize="none"
+          color="transparent"
+          borderRadius="none"
+        >
+          <PolicyEventFiltersList policyId={policyItem.id} />
```

Comment:
advice: pass the whole policy object and not only the id. You will need the policy name later in other tasks.